src/install.rs

```diff
@@ -146,12 +144,14 @@ impl UnixLibNames {
     }
 
     fn links(&self, install_path_lib: &Path) {
-        let mut ln_sf = std::process::Command::new("ln");
-        ln_sf.arg("-sf");
-        ln_sf
-            .arg(&self.with_full_ver)
-            .arg(install_path_lib.join(&self.with_major_ver));
-        let _ = ln_sf.status().unwrap();
+        if self.with_main_ver != self.with_full_ver {
+            let mut ln_sf = std::process::Command::new("ln");
+            ln_sf.arg("-sf");
+            ln_sf
+                .arg(&self.with_full_ver)
+                .arg(install_path_lib.join(&self.with_main_ver));
+            let _ = ln_sf.status().unwrap();
```

Comment:
This panics if the `ln` binary couldn't be executed, and ignores the `ExitStatus` if `ln` returned an error (below for `self.canonical` too).